components/feature/tabs/src/main/java/mozilla/components/feature/tabs/TabsUseCases.kt

```diff
@@ -70,8 +70,20 @@ class TabsUseCases(
         }
     }
 
+    class NotifyOnLowMemoryUseCase internal constructor(private val sessionManager: SessionManager) {
```

Comment:
I don't think this should be a use case. It's not something to be triggered by a user. The session manager is already a component and can directly be notified, see my comment below.